src/workers/jobs.js

```diff
@@ -551,14 +551,21 @@ export async function exportCampaign(job) {
 
       const questionResponses = await r.table('question_response')
         .getAll(contact.id, { index: 'campaign_contact_id' })
+
       Object.keys(allQuestions).forEach((stepId) => {
         let value = ''
         questionResponses.forEach((response) => {
-          if (response.interaction_step_id === stepId) {
+          if (response.interaction_step_id == stepId) {
```

Comment:
I've been fixing this in other places with an explicit `parseInt(...)` on the right side.  some very future date, we might try to move it all to integers (or back to strings!) and having it explicit will probably be good for that moment.

Reply:
response.interaction_step_id is a number
stepId is a string